lib/legato/management/account_summaries.rb

```diff
@@ -0,0 +1,42 @@
+module Legato
+  module Management
+    class AccountSummaries
+
+      extend Finder
+
+      def self.default_path
+        "/accountSummaries"
+      end
+
+      def path
+        "/accountSummaries"
+      end
+
+      attr_accessor :user, :account, :summary_web_properties, :summary_profiles
+
+      def initialize(attributes, user)
+        self.user = user
+
+        self.summary_web_properties = []
+        self.summary_profiles = []
+        self.account = Account.new(attributes, user)
+
+        webProperties = attributes['webProperties'] || attributes[:webProperties]
+
+        for web_property in webProperties
```

Comment:
Please use Ruby conventions here. Underscores for all variable names, and iterators over for loops.